http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/listener/WebSocketSessionManager.java

```diff
@@ -1,106 +0,0 @@
-/*
```

Comment:
This is a previously done change. If it is not appropriate to put it here I can send another PR for this.